### PR TITLE
Remove sequencer gossip

### PR DESCRIPTION
**Pull request by gupadhyaya**
Fixes https://github.com/rollkit/rollkit/issues/781

### CODE REVIEW

File: types/signed_header.go [hunk 1]
```diff
@@ -9,3 +13,24 @@ func (sH *SignedHeader) New() header.Header {
...
+	if !ok {
+		return &header.VerifyError{
+			Reason: fmt.Errorf("%T is not of type %T", untrst, sH),
+		}
+	}
+	return sH.Header.VerifyAdjacent(&untrstH.Header)
+}
+
+func (sH *SignedHeader) VerifyNonAdjacent(untrst header.Header) error {
+	untrstH, ok := untrst.(*SignedHeader)
```
Comment (tuxcanfly) [inline types/signed_header.go:29]: Does the same comment as before `Explicit type checks` apply here?

**Review by tuxcanfly — APPROVED** [review]
OK with minor nit.

File: types/signed_header.go (hunk 1 above)
Comment (gupadhyaya) [reply types/signed_header.go:29]: yes

**Review by nashqueue — APPROVED** [review]
image

**Review by tuxcanfly — APPROVED** [review]
OK.

**Pull request merged**